### PR TITLE
Editor build

### PR DESCRIPTION
**Pull request by humbertodias**
### Editor build

- [x] Migrating from Ant to Gradle
- [x] Local
- [x] GitHub Action

#### 1. Local
Compile
```shell
./easy-compile-editor
```
image

#### 2. GHA

- Build
image

- Release
> [!NOTE]  
> if you create a tag v* the generated artifact will be storage as a release.
image
eg.
https://github.com/humbertodias/allegro-paintown/releases/tag/v1.1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
-build*
+build
```
Comment (kazzmir) [inline .gitignore:1]: I think the changes in this gitignore are not needed

File: editor/Makefile [hunk 2]
```diff
@@ -1,2 +0,0 @@
-all:
-	@-ant
```
Comment (kazzmir) [inline editor/Makefile:2]: I still like having a makefile, even if all it does is invoke `gradlew`

**Comment by kazzmir** [comment]
```
$ java -version
openjdk version "14.0.2" 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-Ubuntu-120.04)
OpenJDK 64-Bit Server VM (build 14.0.2+12-Ubuntu-120.04, mixed mode, sharing)
```
error building:
```
editor $ ./gradlew 
Downloading https://services.gradle.org/distributions/gradle-6.1-bin.zip
.........10%.........20%.........30%..........40%.........50%.........60%..........70%.........80%.........90%.........100%

Welcome to Gradle 6.1!

Here are the highlights of this release:
 - Dependency cache is relocatable
 - Configurable compilation order between Groovy, Java & Scala
 - New sample projects in Gradle's documentation

For more details see https://docs.gradle.org/6.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
java.lang.NoClassDefFoundError: Could not initialize class org.codehaus.groovy.vmplugin.v7.Java7
        at org.codehaus.groovy.vmplugin.VMPluginFactory.<clinit>(VMPluginFactory.java:43)
```

**Comment by humbertodias** [comment]
Upgrading gradlew version to 7.6 to be compatible with your java 14
```
Gradle 7.x: Java 11, Java 16 (experimental)
Gradle 6.x: Java 8, Java 11
Gradle 5.x: Java 8, Java 11
Gradle 4.x: Java 6, Java 7, Java 8
Gradle 3.x: Java 6, Java 7, Java 8
Gradle 2.x: Java 6, Java 7, Java 8
Gradle 1.x: Java 5, Java 6, Java 7
```

**Comment by humbertodias** [comment]
Recreated

**Pull request merged**